supervisor/testing_analyst.py

```diff
@@ -58,13 +59,16 @@ class OutputSchema(BaseModel):
     + """
 For components handled by the New Errata Workflow Automation(NEWA):
 NEWA will post a comment to the erratum when it has started tests and when they finish.
-Read the JIRA issue in those comments to find test results. Ignore any comments
-with links to TCMS or Beaker; older EWA automation may have run in parallel
-with NEWA, but should be ignored.
+Read the JIRA issue in those comments to find test results.
+For components handled by Errata Workflow Automation (EWA):
+EWA will post a comment to the erratum when it has started tests and when they finish.
+Read the comment to find the test results in TCMS Test Run.
+If there are comments from both NEWA and ANEWA, ignore the comments with links to Testing  Farm
+ older EWA automation have run in parallel with NEWA,  and provides old-new comparison.
```

Comment:
I think you meant EWA not ANEWA? My preference here would be to prefer the NEWA tests in the case where both occur: it's the modern one not the one being slowly replaced, and while EWA is better in providing the old/new comparison, we have a solution for that.

So I'd say something like:

```suggestion

If the test location data says that tests are started by NEWA, but there are no comments
from NEWA providing links to JIRA issues, then this component may be a component where
NEWA is only used for RHEL10, and not earlier versions - in that case, you may read the
results from the TCMS test run posted by EWA.

In all other cases, if the tests are supposed to be started by NEWA, ignore any comments with
links to TCMS or Beaker.
```